swarm/storage/localstore/localstore.go

```diff
@@ -29,6 +29,9 @@ import (
 	"github.com/ethereum/go-ethereum/swarm/storage/mock"
 )
 
+// DB implements chunk.Store.
+var _ chunk.Store = &DB{}
```

Comment:
?

Reply:
A static check at compile time that DB is implementing all chink store methods.